common/Fabric-Install-Utilities.psm1

```diff
@@ -414,10 +414,25 @@ function Get-CurrentScriptDirectory()
     return Split-Path $script:MyInvocation.MyCommand.Path
 }
 
-function Get-InstallationSettings($configSection)
+function Get-InstallationSettings
 {
-    $installationConfig = [xml](Get-Content install.config)
-    $sectionSettings = $installationConfig.installation.settings.scope | where {$_.name -eq $configSection}
+    param(
+        [Parameter(Mandatory=$true)]
+        [string] $configSection,
+        [ValidateScript({
+            if (!(Test-Path $_)) {
+                throw "Path $_ does not exist. Please enter valid path to the install.config."
+            }
+            if (!(Test-Path $_ -PathType Leaf)) {
+                throw "Path $_ is not a file. Please enter a valid path to the install.config."
+            }
+            return $true
+        })]  
+        [string] $installConfigPath = "install.config"
+    )
+
+    $installationConfig = [xml](Get-Content $installConfigPath)
+    $sectionSettings = $installationConfig.installation.settings.scope | Where-Object {$_.name -eq $configSection}
     $installationSettings = @{}
 
     foreach($variable in $sectionSettings.variable){
```

Comment:
Do we want to log if `$variable.name` or `$variable.value` is missing?

Reply:
I'm trying to minimize changes to this module because we will eventually move away from it. Just attempting to get the support in for quiet installs.